packages/store-cache/src/createDatabaseClient.ts

```diff
@@ -23,6 +22,7 @@ export function createDatabaseClient<C extends StoreConfig>(database: TupleDatab
 
   // Create utils with client, namespace and table argument prefilled
   for (const table in config.tables) {
+    const namespace = config.tables?.[table].namespace ?? config.namespace;
```

Comment:
would the table config's namespace ever be undefined? since put the default namespace there while expanding the config?

Reply:
no, namespace is always set by zod.
`config.tables?.[table]` the ? confused me a bit into thinking the whole thing could be undefined